packages/inventory/src/Stocktake/api/api.ts

```diff
@@ -1,5 +1,5 @@
-import { DraftStocktakeLine } from './../DetailView/modal/StocktakeLineEdit/hooks';
 import {
+  generateUUID,
```

Comment:
little bit of reorganising.. think it's getting to a nice place..?

Reply:
sure is.. looking very nice